code/modules/clothing/suits/armor.dm

```diff
@@ -94,8 +96,8 @@
 	icon_state = "riot"
 	item_state = "swat"
 	flags_armor_protection = CHEST|GROIN|LEGS|ARMS
-	slowdown = 1.2
-	soft_armor = list(MELEE = 65, BULLET = 10, LASER = 10, ENERGY = 10, BOMB = 0, BIO = 0, FIRE = 10, ACID = 10)
+	slowdown = 0.9
+	soft_armor = list(MELEE = 75, BULLET = 15, LASER = 10, ENERGY = 10, BOMB = 0, BIO = 10, FIRE = 10, ACID = 30)
```

Comment:
Make a subtype of these instead of buffing the base armors. You can get 3 of these for the price of a b18 in req. All items in this file.